include/mbedtls/config.h

```diff
@@ -58,6 +58,20 @@
  */
 #define MBEDTLS_HAVE_ASM
 
+/**
+ * \def MBEDTLS_MPI_FORCE_C_CODE
+ *
+ * The compiler should compile C version of MPI code, even when the compiler
+ * supports asm(). This allows it to automatically insert (speculative execution)
+ * countermeasures.
```

Comment:
reword: This enables the use of C compiler passes, such as automatic insertion of transient execution attack countermeasures.